source/get-started/download-and-install.txt

```diff
@@ -0,0 +1,71 @@
+.. _scala-quick-start-download-and-install:
+
+====================
+Download and Install
+====================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: installation, setup, code example
+
+.. procedure::
+   :style: connected
+
+   .. step:: Install dependencies
+
+      Before you being developing, ensure you have the following dependencies
+      installed in your development environment:
+      
+      - `JDK <https://www.oracle.com/java/technologies/downloads/>`__ version 8 or later
+      - `sbt <https://www.scala-sbt.org/1.x/docs/Setup.html>`__ version 1 or later
+
+   .. step:: Create a project directory
+
+      Run the following command in your shell to create a directory
+      called ``scala-quickstart`` for this project:
+
+      .. code-block:: bash
+
+         mkdir scala-quickstart
+
+      Select the tab corresponding to your operating system and run the following commands
+      to create a ``build.sbt`` file in the ``scala-quickstart`` directory:
+
+      .. tabs::
+
+          .. tab:: macOS / Linux
+             :tabid: create-file-mac-linux
+
+             .. code-block:: bash
+
+                cd scala-quickstart
+                touch build.sbt
+
+          .. tab:: Windows
+             :tabid: create-file-windows
+
+             .. code-block:: bash
+
+                cd scala-quickstart
+                type nul > build.sbt
+
+   .. step:: Configure your project to use the {+driver-short+}
+
+      Navigate to your ``build.sbt`` file and add the following code to use
+      the {+driver-short+} in your application:
+
+      .. code-block:: none
+         
+         ThisBuild / scalaVersion := "2.13.12"
```

Comment:
If the scala version number here might change in the future we should use a source constant for it (also applies to the version below this)